test/functional/shared.js

```diff
@@ -204,7 +204,7 @@ function withMonitoredClient(commands, callback) {
     client.connect((err, client) => {
       expect(err).to.not.exist;
       const d = function() {
-        const args = Object.values(arguments);
+        const args = Object.keys(arguments).map(e => arguments[e]);
```

Comment:
isn't the expectation that `done` is always called with either `err` or `undefined`? why can't do we just do:
```
function _done(err) {
  client.close(err2 => done(err || err2));
}

callback.bind(this).(client, events, _done);
```

Reply:
Yeah this rocks!